crates/ruff/src/logging.rs

```diff
@@ -171,32 +171,29 @@ impl Display for DisplayParseError<'_> {
         // If we're working on a Jupyter notebook, translate the positions
         // with respect to the cell and row in the cell. This is the same
         // format as the `TextEmitter`.
-        let error_location = if let Some(jupyter_index) = self
-            .source_kind
-            .and_then(SourceKind::notebook)
-            .map(Notebook::index)
-        {
-            write!(
-                f,
-                "cell {cell}{colon}",
-                cell = jupyter_index
-                    .cell(source_location.row.get())
-                    .unwrap_or_default(),
-                colon = ":".cyan(),
-            )?;
+        let error_location =
+            if let Some(jupyter_index) = self.source_kind.as_ipy_notebook().map(Notebook::index) {
```

Comment:
This is the main change in this section of the diff otherwise it's just a formatter change.